app/assets/javascripts/application.js

```diff
@@ -71,6 +71,15 @@ $(document).on('railsAutocomplete.select', '#fake_searched_id', function(){
 
 $(document).ready(function() {
 
+  // Logo Swap
```

Comment:
So this is still a JS swap, not a CSS swap? That was the issue that I ran into before; while having an inline SVG allowed us to manipulate its colors using CSS / JS, I couldn't figure out how to do it with just CSS because of the way that Illustrator exported the SVG file. In retrospect, this is still cleaner than swapping files, so it's probably the best solution.